packages/nbdime/src/common/mergeview.ts

```diff
@@ -409,6 +421,92 @@ const conflictMarkerLineChunkMappingField = StateField.define<
   },
 });
 
+/**
+ * Uncollapse effect to synchronize uncollapsed ranges between editors.
+ */
+const uncollapse = StateEffect.define<number>({
+  map: (value, change) => change.mapPos(value),
+});
+
+/**
+ * Effect to set the collapser widgets in an editor
+ */
+const setCollapsers = StateEffect.define<DecorationSet>({
```

Comment:
```suggestion
 */
const setCollapsersEffect = StateEffect.define<DecorationSet>({
```
All effects have `Effect` as a suffix.